buildenv/jenkins/common/build.groovy

```diff
@@ -370,13 +383,19 @@ def archive_sdk() {
                         currentBuild.description += "<br><a href=${JAVADOC_LIB_URL}>${JAVADOC_FILENAME}</a>"
                         echo "Javadoc:'${JAVADOC_LIB_URL}"
                     }
+                    if (fileExists("${JAVADOC_OPENJ9_ONLY_FILENAME}")) {
+                        JAVADOC_OPENJ9_ONLY_LIB_URL = "${ARTIFACTORY_CONFIG[ARTIFACTORY_CONFIG['defaultGeo']]['url']}/${ARTIFACTORY_CONFIG['uploadDir']}${JAVADOC_OPENJ9_ONLY_FILENAME}"
+                        currentBuild.description += "<br><a href=${JAVADOC_OPENJ9_ONLY_LIB_URL}>${JAVADOC_OPENJ9_ONLY_FILENAME}</a>"
+                        echo "Javadoc (OpenJ9 extensions only):'${JAVADOC_OPENJ9_ONLY_LIB_URL}"
```

Comment:
As I had another look at this today, I noticed that this message is missing the closing single quote just like line 384: could you fix both, please?